front-end/js/jump.js

```diff
@@ -362,6 +390,147 @@ function registerJumps(viewId, svg, layerId) {
         // make cursor a hand when hovering over this shape
         d3.select(this).style("cursor", "zoom-in");
 
+        // disable context menu on kyrix objects since we use right clicks for update
```

Comment:
Note what the equivalent is for mac (and mobile devices, if supported...?)

Reply:
I've tested this using an external mouse and with the mac trackpad. I'm definitely assuming that we aren't supporting mobile devices because that hasn't been a goal of Kyrix so far.

Reply:
Great point by Adam. 

Note that currently Kyrix supports mobile out of the box. It' not clear what the equivalent of right click on mobile is.

Reply:
Just commenting here so I remember that we will now make the update option part of the jump menu. I think everyone was in agreement with that since it makes everything peachy on mobile, even though i don't expect kyrix users to want to edit a vis on mobile but who's knows maybe on ipad